public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -316,7 +316,9 @@ block add-new-hero-via-detail-comp
   The user can *delete* an existing hero by clicking a delete button next to the hero's name.
   Add the following to the heroes component HTML right after the hero name in the repeated `<li>` tag:
 +makeExample('app/heroes.component.html', 'delete')
-
+:marked
+  Add the following CSS to the bottom of `heroes.component.css`.
++makeExcerpt('toh-6/ts/app/heroes.component.css', 'additions', '')(format=".")
```

Comment:
Shorten the path to just `app/heroes.component.css` (just like for line 318 above).
Drop the `(format=".")` since that is the default for `makeExcerpt`.